Router container

pgoulding opened the pull request:
This is mostly just fixing the styling for the card container. Now the cards are stacking and responsive, we might want to work on making the buttons more responsive as well. Maybe a flex wrap, or have them become smaller when the cards appear? Then probably need a media query and we should be pretty good with the styling.

TaylorNoelJordan commented:
Yeah I had a significant bit of this working that seems to be broken now... It was tied to setting the state of the app component. I had a selectCategory method that has been deleted. Could you shed some light on that so I can trace a path to fixing it?

pgoulding commented:
Select Category was a method I had for the fetch calls whenever you clicked a different category and assigning it to state.data, deleted it yesterday when we moved to mapping over the categories and setting them to state[category]. I don't recall deleting any of your methods attached to that, but maybe it was a merge conflict when pushing to master? We can make a new method for setting your selected button animation. Looked through some of the commits yesterday but not sure if I'm missing anything for when the animation fires. DM me on slack and we can work out a solution!
